rsmtool/analysis.py

```diff
@@ -448,9 +555,34 @@ def compute_metrics_by_group(df_test,
                              include_second_score=False):
     """
     Compute a subset of the evaluation metrics for the scores
-    in the given data frame by group specified in grouping variables.
+    in the given data frame by group specified in `grouping_variable`.
+
     See `filter_metrics()` above for a description of the subset
     that is selected.
+
+    Parameters
+    ----------
+    df_test : pandas DataFrame
+        Input data frame.
+    grouping_variable : str
+        Feature name indicating the column that
+        contains grouping information.
+    use_scaled_predictions : bool, optional
+        Include scaled predictions when computing
```

Comment:
"correlations statistics" -> "evaluation metrics" (not just correlation. Scaling has no effect on correlations anyway)